apps/myaccount/java/org.wso2.identity.apps.myaccount.server.feature/resources/deployment.config.json.j2

```diff
@@ -389,5 +389,8 @@
         "theme": {
             "name": "{{ myaccount.theme }}"
         }
-    }
+    }{{ "," if myaccount.validate_id_token_issuer is defined }}
```

Comment:
Let's add this under `idpConfigs`

Reply:
Since this new config enables/ disbales id token issuer validation, it does not belong to `idpConfigs`.

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/5050/commits/4e0f4328aad4466579342d39c0d658788c7a6b1b